README.md

```diff
@@ -49,27 +49,22 @@ If the verification code is expired or does not match the user's email address,
 ## Config settings
 
 ### Length
-The length of the verification codes. The standard length for a verification code is `6`.
+This value defines the length of every generated verification code.
 
-### Type
-The character set the package will use to generate verification codes. The supported character sets are `numeric`, `alphabetical` and `alphanumeric`. The `numeric` character set is used by default.
-
-### Exclude characters
-You sometimes may want to exclude one or more characters from the selected or default character set. This can be done simply by adding the character(s) to the `exclude_characters` array in the config (e.g. `excluded_characters => ['0', '9']`, verification codes will be issued between 1 and 8).
+### Characters
+You can define which characters are used to generate a verification code. By default, certain characters are excluded (0, O, I, L) because they look to similar.
```

Comment:
```suggestion
You can define which characters are used to generate a verification code. By default, certain characters are excluded (0, O, I, L) because they look too similar.
```